khmer/_khmermodule.cc

```diff
@@ -23,7 +23,7 @@
 #include "labelhash.hh"
 #include "khmer_exception.hh"
 #include "hllcounter.hh"
-
+//#include <stdlib.h>
```

Comment:
Please undo this change. Try to check your pull requests before requesting review either via the GitHub interface ( https://github.com/ged-lab/khmer/pull/771/files ) or via `git fetch; git diff origin/master`